README.md

```diff
@@ -51,6 +51,14 @@ services:
 
 To test, run `docker-compose up`.
 
+## Unit Tests
+
+We use the Jest unit testing framework for some simple testing. To run the unit tests, run the following command at the root of the repo:
+
+```
+npm run test
```

Comment:
Hey Jesse, good call out! I'm not having that SyntaxError (in fact, when I run `docker run opentransit-collector npm run test`, I get no output at all). Where are you seeing that error?

Reply:
```
$ docker build -t opentransit-collector .
[+] Building 1.6s (10/10) FINISHED
 => [internal] load build definition from Dockerfile                                                               0.0s
 => => transferring dockerfile: 32B                                                                                0.0s
 => [internal] load .dockerignore                                                                                  0.0s
 => => transferring context: 34B                                                                                   0.0s
 => [internal] load metadata for docker.io/library/node:8.6.0                                                      1.3s
 => [1/5] FROM docker.io/library/node:8.6.0@sha256:a8f7411582dd13c0f3cb2b168e89bc8ab2772553665b8269ba275cadaca38b  0.0s
 => [internal] load build context                                                                                  0.0s
 => => transferring context: 4.15kB                                                                                0.0s
 => CACHED [2/5] WORKDIR /usr/src/app                                                                              0.0s
 => CACHED [3/5] COPY package.json package-lock.json ./                                                            0.0s
 => CACHED [4/5] RUN npm install                                                                                   0.0s
 => [5/5] COPY . .                                                                                                 0.1s
 => exporting to image                                                                                             0.1s
 => => exporting layers                                                                                            0.0s
 => => writing image sha256:93f2f4bc58b9ab2991ab69c9a2cab1ed237d5a7a418f8905ca89f02f213f621f                       0.0s
 => => naming to docker.io/library/opentransit-collector                                                           0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them

$ docker run opentransit-collector npm run test
npm info it worked if it ends with ok
npm info using npm@5.3.0
npm info using node@v8.6.0
npm info lifecycle opentransit-collector@1.1.0~pretest: opentransit-collector@1.1.0
npm info lifecycle opentransit-collector@1.1.0~test: opentransit-collector@1.1.0

> opentransit-collector@1.1.0 test /usr/src/app
> jest

/usr/src/app/node_modules/jest-cli/build/cli/index.js:219
    } catch {
            ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:588:28)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/src/app/node_modules/jest-cli/build/index.js:13:12)
npm info lifecycle opentransit-collector@1.1.0~test: Failed to exec test script
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! opentransit-collector@1.1.0 test: `jest`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the opentransit-collector@1.1.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-02-19T17_25_56_793Z-debug.log
```

Reply:
Hmm yeah for some reason the output of the run doesn't show up in stdout, I have to run `docker logs` to see it. Weird.

In any case, I pushed up using the latest Node LTS version, in case we want to play it safe on breaking changes. The syntax error is no longer present. Though now when running collector this error is popping up 

```
collector-dev  | { RequestTimeTooSkewed: The difference between the request time and the current time is too large.
collector-dev  |     at Request.extractError (/usr/src/app/node_modules/aws-sdk/lib/services/s3.js:710:35)
collector-dev  |     at Request.callListeners (/usr/src/app/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
collector-dev  |     at Request.emit (/usr/src/app/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
collector-dev  |     at Request.emit (/usr/src/app/node_modules/aws-sdk/lib/request.js:686:14)
collector-dev  |     at Request.transition (/usr/src/app/node_modules/aws-sdk/lib/request.js:22:10)
collector-dev  |     at AcceptorStateMachine.runTo (/usr/src/app/node_modules/aws-sdk/lib/state_machine.js:14:12)
collector-dev  |     at /usr/src/app/node_modules/aws-sdk/lib/state_machine.js:26:10
collector-dev  |     at Request.<anonymous> (/usr/src/app/node_modules/aws-sdk/lib/request.js:38:9)
collector-dev  |     at Request.<anonymous> (/usr/src/app/node_modules/aws-sdk/lib/request.js:688:12)
collector-dev  |     at Request.callListeners (/usr/src/app/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
collector-dev  |   message: 'The difference between the request time and the current time is too large.',
collector-dev  |   code: 'RequestTimeTooSkewed',
collector-dev  |   region: null,
collector-dev  |   time: 2022-02-20T01:49:30.315Z,
collector-dev  |   requestId: 'E7K9TJAAMWN60SW5',
collector-dev  |   extendedRequestId: 'C1ldAFNXPM/Mpv/4GZfaPmKsdiSkXgPLl/5bTx5z9e0fN2r+WHFhVXS2DA/ddWllSay0SR3Svv4=',
collector-dev  |   cfId: undefined,
collector-dev  |   statusCode: 403,
collector-dev  |   retryable: false,
collector-dev  |   retryDelay: 7.329905100921441 }
```

Have you seen that before? I doubt this is from the Node update, feels like something else is going on

Reply:
Came back to look at this and all is well :shrug: on a different OS so maybe I had a weird configuration on my end. Marking this as resolved, but let me know if there's something else to do in terms of the docker / npm setup.

Reply:
Are you using Docker on Windows 10/11? In that case, you might be able to fix the time in the Docker containers by running `wsl hwclock -s` from a windows command prompt. This will synchronize the VM clock with the Windows system clock.